### PR TITLE
AA/Bundler documentation

### LINKED ISSUE #737: Document Bundler SDK for docs.boba.network
https://github.com/bobanetwork/boba/pull/698

just some more info - on the same branch ^ - its mostly these two folders
https://github.com/bobanetwork/boba/tree/inomurko/bump-bundler-2/packages/boba/bundler_sdk
https://github.com/bobanetwork/boba/tree/inomurko/bump-bundler-2/packages/boba/bundler
if you look at bundler_sdk's readme - its quite outdated + we need to add docs about the changes we have made to the sdk in order for users to use it
this is the precise upstream commits the latest is synced with
contracts: https://github.com/eth-infinitism/account-abstraction/commit/b18e2333d9f9a0989176382b923457d672d6fcac
bundler: https://github.com/eth-infinitism/bundler/commit/366c7b65af9be8a31f18db939ee05ee298c086cb

reference how docs should look like: https://docs.infura.io/infura/networks/ethereum/json-rpc-methods
sdk: https://pkg.go.dev/crypto/rand#pkg-functions
published on docs.boba.network

### PR DESCRIPTION
**Pull request by wsdt**
## Overview

Add documentation for bundler. 

Closes #737 , #738 

## Changes

- Adding new documentation to content list
- Adding new pages for bundler documentation
- Bundler SDK docs
- Bundler API docs

### CODE REVIEW

**Comment by InoMurko** [comment]
Very good!

**Review by InoMurko — APPROVED** [review]

**Comment by wsdt** [comment]
@souradeep-das not 100% sure about the error codes for the Bundler API.

**Comment by InoMurko** [comment]
When time comes, this will need to be updated with URLs to the Bundler on different networks

**Comment by souradeep-das** [comment]
Thanks Kevin! I'll append this shortly with the sdk side changes (to some methods) introduced to make it work on Boba.
will double check the error codes too

**Comment by souradeep-das** [comment]
added a commit for the additional conditions to use the sdk
please review this commit [c3851e4](https://github.com/bobanetwork/boba/pull/736/commits/c3851e4d8ff3729fc05e460bd84687b630bed50a)

File: package-lock.json
Comment (souradeep-das) [inline package-lock.json:1]: probably a leftover?

**Review by souradeep-das — APPROVED** [review]

File: package-lock.json
Comment (wsdt) [reply package-lock.json:1]: removed thx!

**Pull request merged**